PHPUnit/Extensions/Selenium2TestCase.php

```diff
@@ -166,6 +166,11 @@
     /**
      * @param boolean
      */
+    private static $keepSeesionOnFail = FALSE;
```

Comment:
There is a typo in this variable, it should be `keepSessionOnFail`? Better yet `keepSessionOnFailure`?